.circleci/verbatim-sources/pytorch-job-specs.yml

```diff
@@ -19,6 +19,27 @@ jobs:
           docker pull ${DOCKER_IMAGE} >/dev/null
           export id=$(docker run -t -d -w /var/lib/jenkins ${DOCKER_IMAGE})
 
+          # Rebase to master only if in xenial_py3_6_gcc5_4 case
```

Comment:
Better to put this chunk of code into a "step"?

Reply:
Sure, if that is preferred, I can turn it into a step.